### PR TITLE
Feat/material design

### LINKED ISSUE #26: New Materialized Design
We have a clean, new design that should be used for the map.

refugee-map-screenflow

Screenflow and design is up here: https://drive.google.com/a/ubilabs.net/folderview?id=0B5DexgyviHntQzBJbFh3RFVudmM&usp=sharing

### PR DESCRIPTION
**Pull request by ro-ka**
Closes #26

### CODE REVIEW

**Comment by ro-ka** [comment]
#26

File: assets/scripts/views/actions.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+ */
+export default class {
+  /**
+   * Initialize
+   */
+  constructor({
+    onMenuToggle = () => {},
+    onFiltersToggle = () => {},
+    onCenter = () => {}
+  }) {
```
Comment (aemkei) [inline assets/scripts/views/actions.js]: Are these default handlers, that will be overridden? It looks a bit strange here, cause of the fat arrow syntax.

Comment (aemkei) [inline assets/scripts/views/actions.js]: Aah, I see: These are the default options pointing to an empty handler using an object spread.

Comment (ro-ka) [inline assets/scripts/views/actions.js]: Exactly. :)

**Comment by ro-ka** [comment]
Updated. Ready now. Will create new issues for the missing parts.

File: assets/scripts/views/filters.js [hunk 2]
```diff
@@ -1,53 +1,86 @@
...
-    categories.forEach(category => {
-      let $categoryFilter = document.createElement('div'),
-        $categoryFilterIcon = document.createElement('img'),
-        $categoryFilterText = document.createElement('span');
+    categories.unshift({
+      key: 'all',
+      visible: 'y',
+      english: 'All categories',
+      german: 'Alle Kategorien',
+      arabic: 'جميع الفئات'
```
Comment (aemkei) [inline assets/scripts/views/filters.js]: Would be great to have one place for all the translations. See: #30

Comment (aemkei) [inline assets/scripts/views/filters.js]: PS: We can improve that later.

**Pull request merged**